pkg/cmd/kitimport/kitimport.go

```diff
@@ -204,3 +205,25 @@ func promptToEditKitfile(contextDir, tag string, currentKitfile *artifact.KitFil
 	}
 	return readExistingKitfile(kitfilePath)
 }
+
+func getEditorName() (string, error) {
+	// Default: use the editor in the standard $EDITOR environment variable
+	editor := os.Getenv("EDITOR")
+	if editor != "" {
+		return editor, nil
+	}
+	if runtime.GOOS == "linux" || runtime.GOOS == "darwin" {
+		// Check for nano, which should be the default editor on Linux and MacOS
+		checkCommand := exec.Command("command", "-v", "nano")
```

Comment:
Is this essentially doing `exec.LookPath("nano")`?

Reply:
Right, forgot that existed :) 

Fixed